include/dlr.h

```diff
@@ -88,7 +106,7 @@ typedef struct DLR_TFTensorDesc {
 int CreateDLRModelFromTensorflow(DLRModelHandle* handle, const char* model_path,
                                  const DLR_TFTensorDesc* inputs, int input_size,
                                  const char* outputs[], int output_size,
-                                 const int threads);
+                                 const DLR_TFConfig tf_config);
```

Comment:
`const DLR_TFConfig tf_config` -> `const DLR_TFConfig& tf_config`

Reply:
Since it is C header we can pass by value or use pointer.